src/Microsoft.Extensions.Options.ConfigurationExtensions/ConfigurationChangeTracker.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.Extensions.OptionsModel
+{
+    public class ConfigurationChangeTracker<TOptions> : IOptionsChangeTracker<TOptions>
+    {
+        private IConfiguration _config;
+
+        public ConfigurationChangeTracker(IConfiguration config)
```

Comment:
Well we bind against IConfiguraiton usually for options, felt more convenient to be able to take any part of config (since internally they know how to get to the root anyways).  This way you can be passed the relevant section (if we wanted to have an overload for Bind + TrackChanges in one call, i.e. `Configure<MyOptions>(config.Get("My", trackChanges: true)`